recipes/remix/app/root.tsx

```diff
@@ -7,13 +7,21 @@ import {
   Outlet,
   Scripts,
   ScrollRestoration,
+  // useMatches,
 } from "@remix-run/react";
 
 export const links: LinksFunction = () => [
   ...(cssBundleHref ? [{ rel: "stylesheet", href: cssBundleHref }] : []),
 ];
 
 export default function App() {
+  /**
+   * Disable client-side JS - Optional
+   * @see https://remix.run/docs/en/main/guides/disabling-javascript
+   */
+  // const matches = useMatches();
+  // const includeScripts = matches.some((match) => match.handle?.hydrate);
+
```

Comment:
I'm a bit 50/50 about whether we should include this. It's great, but also bloats the recipe for users that want JS.

Let me have a think on it before we remove it.